src/js/player.js

```diff
@@ -549,6 +550,25 @@ class Player extends Component {
       this.addClass('vjs-touch-enabled');
     }
 
+    if (options.useSVGIcons) {
+      // Add SVG Sprite to the DOM
+      const parser = new window.DOMParser();
+      const parsedSVG = parser.parseFromString(icons, 'image/svg+xml');
+      const errorNode = parsedSVG.querySelector('parsererror');
+
+      if (errorNode) {
+        log.warn('Failed to load SVG Icons. Falling back to Font Icons.');
+        return;
+      }
```

Comment:
Good idea here. Should this also set the player option to `false` so that other code doesn't think SVG icons are disabled?

Reply:
Great call! I reset the option, updated the logic to not return if the failure happened, as well as make sure this all happens before the children components are initialized: https://github.com/videojs/video.js/commit/357adf1b267515534e4253af49a46a59777fdc1d#diff-f92dc431786299ba52ba41d6901e029cc1eeb2f5603653bd03eb4d88a27f5020R521